### ISSUE TITLE
Uploading records to second database broken in current vagrant version

### ISSUE THREAD

**Issue by sneumann** [MassBank/MassBank-web#31]
```
org.apache.jasper.JasperException: An exception occurred processing JSP page /mbadmin/RecordRegist.jsp at line 551

548: 				pw[i] = new PrintWriter( new BufferedWriter(new FileWriter(filePath)) );
549: 				pw[i].println( "START TRANSACTION;");
550: 			}
551: 			SqlFileGenerator sfg = new SqlFileGenerator( baseUrl, selDbName, ver );
552: 			for ( int i=0; i<dataFileList.size(); i++ ) {
553: 				String filePath = registPath + File.separator + dataFileList.get(i);
554: 				sfg.readFile( filePath );


Stacktrace:
	org.apache.jasper.servlet.JspServletWrapper.handleJspException(JspServletWrapper.java:568)
	org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:470)
	org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:395)
	org.apache.jasper.servlet.JspServlet.service(JspServlet.java:339)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:727)

root cause

java.lang.ArrayIndexOutOfBoundsException: 1
	massbank.GetInstInfo.getInformation(GetInstInfo.java:109)
	massbank.GetInstInfo.<init>(GetInstInfo.java:49)
	massbank.admin.SqlFileGenerator.<init>(SqlFileGenerator.java:77)
	org.apache.jsp.mbadmin.RecordRegist_jsp.registRecord(RecordRegist_jsp.java:540)
	org.apache.jsp.mbadmin.RecordRegist_jsp._jspService(RecordRegist_jsp.java:1175)
	org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:727)
	org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:432)
	org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:395)
	org.apache.jasper.servlet.JspServlet.service(JspServlet.java:339)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:727)
```

**Comment by sneumann** [comment]
This one happens in https://github.com/MassBank/MassBank-web/blob/master/modules/tomcat/webapps/MassBank/mbadmin/RecordRegist.jsp#L551 where some steps down in 
https://github.com/MassBank/MassBank-web/blob/6a4f6595e4658705624cf76e4e65312d7f945392/Tomcat/webapps/ROOT/WEB-INF/src/massbank/GetInstInfo.java#L80 `getInformation()` is called, 
which (similar to #30) calls `execDispatcher()` to retrieve some list, which is expected to be tab-separated, and for an isInst (=isInstrument ?) should contain three items,
number, type and name, but it does not.

I thought, instrument information is obtained by calling http://192.168.35.18/MassBank/cgi-bin/GetInstInfo.cgi or http://192.168.35.18/MassBank/cgi-bin/GetInstInfo.cgi?ver=2, maybe here is 
a mismatch. 


 
`

**Comment by tsufz** [comment]
I installed the new server into a local VM and added a new DB, no problems. Then I tried to upload new files (using the example recdata.zip). The old bug still occurs, same using only one DB.

HTTP Status 500 - An exception occurred processing JSP page /mbadmin/RecordRegist.jsp at line 551

type Exception report

message An exception occurred processing JSP page /mbadmin/RecordRegist.jsp at line 551

description The server encountered an internal error that prevented it from fulfilling this request.

exception

org.apache.jasper.JasperException: An exception occurred processing JSP page /mbadmin/RecordRegist.jsp at line 551

548: 				pw[i] = new PrintWriter( new BufferedWriter(new FileWriter(filePath)) );
549: 				pw[i].println( "START TRANSACTION;");
550: 			}
551: 			SqlFileGenerator sfg = new SqlFileGenerator( baseUrl, selDbName, ver );
552: 			for ( int i=0; i<dataFileList.size(); i++ ) {
553: 				String filePath = registPath + File.separator + dataFileList.get(i);
554: 				sfg.readFile( filePath );


Stacktrace:
	org.apache.jasper.servlet.JspServletWrapper.handleJspException(JspServletWrapper.java:578)
	org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:476)
	org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:401)
	org.apache.jasper.servlet.JspServlet.service(JspServlet.java:345)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
	org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)

root cause

java.lang.ArrayIndexOutOfBoundsException: 1
	massbank.GetInstInfo.getInformation(GetInstInfo.java:109)
	massbank.GetInstInfo.<init>(GetInstInfo.java:49)
	massbank.admin.SqlFileGenerator.<init>(SqlFileGenerator.java:77)
	org.apache.jsp.mbadmin.RecordRegist_jsp.registRecord(RecordRegist_jsp.java:541)
	org.apache.jsp.mbadmin.RecordRegist_jsp._jspService(RecordRegist_jsp.java:1258)
	org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
	org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:438)
	org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:401)
	org.apache.jasper.servlet.JspServlet.service(JspServlet.java:345)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
	org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)

note The full stack trace of the root cause is available in the Apache Tomcat/8.0.32 (Ubuntu) logs.
Apache Tomcat/8.0.32 (Ubuntu)

**Comment by Treutler** [comment]
fixed by commit 9e227da4fedcfdda00da4de38c546a22d276c894
